### PR TITLE
Des 554

### PR DESCRIPTION
**Pull request by briandunnpearson**
Using the component-archetype style, more or less.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,22 @@
 language: node_js
 sudo: false
 node_js:
+- 4
```
Comment (mhomolak) [inline .travis.yml]: I think we can remove 4 since we're only going to be testing 6 & 8 now.

File: demo/demo.js [hunk 2]
```diff
@@ -1,131 +1,36 @@
...
     opts: {
-      title: 'Coach Mark below feature /w Got It',
-      text: 'Some text explaining to the user why you changed their interface',
+      title: 'Coach Mark Above',
+      text: 'Consumer passes in .forceAbove, .disableShadowing, and .gotIt',
       gotIt: true,
-      disableShadowing: true,
-      id: '9834893449'
-    },
-    callback: function (id) {
```
Comment (mhomolak) [inline demo/demo.js]: No more callback hell!

File: .travis.yml [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 - 6
-- 4.2
+- 8
 script:
 - npm run ci
-- chmod 777 ./node_modules/@pearson-components/npm-scripts/shell_scripts/run_selenium_ci_tests.sh
-- export TEST_GROUP=desktop-regression
```
Comment (mhomolak) [inline .travis.yml]: We need to leave this line.

https://github.com/Pearson-Higher-Ed/npm-scripts/blob/master/shell_scripts/run_selenium_ci_tests.sh#L27

File: test/shell_scripts/run_tests.sh [hunk 4]
```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```
Comment (mhomolak) [inline test/shell_scripts/run_tests.sh]: We don't need this file or the shell_scripts folder anymore.  This file should be ran through the npm-script node module now.

**Comment by briandunnpearson** [comment]
Ready for QA

**Comment by udhadpa** [comment]
Tests updated and approved. Good to merge

**Pull request merged**